docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToImage.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ConvertToImage
+    {
+        private const int imageHeight = 224;
+        private const int imageWidth = 224;
+        private const int numberOfChannels = 3;
+        private const int inputSize = imageHeight * imageWidth * numberOfChannels;
+
+        // Sample that shows how an input array (of doubles) can be used to interop with image related estimators in ML.NET.
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create a list of training data points.
+            var dataPoints = GenerateRandomDataPoints(4);
+
+            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            var data = mlContext.Data.LoadFromEnumerable(dataPoints);
+
+            // Image loading pipeline. 
+            var pipeline = mlContext.Transforms.ConvertToImage(imageHeight, imageWidth, "Image", "Features")
+                          .Append(mlContext.Transforms.ExtractPixels("Pixels", "Image"));
+
+            var transformedData = pipeline.Fit(data).Transform(data);
+
+            // Preview the transformedData. 
+            var transformedDataPreview = transformedData.Preview();
+            PrintPreview(transformedDataPreview);
+            // Features                 Image                    Pixels
+            // 185,209,196,142,52       System.Drawing.Bitmap    185,209,196,142,52
+            // 182,235,84,23,87         System.Drawing.Bitmap    182,235,84,23,87
+            // 192,214,247,22,38        System.Drawing.Bitmap    192,214,247,22,38
+            // 242,161,141,223,192      System.Drawing.Bitmap    242,161,141,223,192
+        }
+
+        private static void PrintPreview(DataDebuggerPreview data)
+        {
+            foreach (var colInfo in data.ColumnView)
+                Console.Write("{0, -25}", colInfo.Column.Name);
```

Comment:
>"{0, -25}" [](start = 30, length = 10)

let's  just define this as:
var format = "{0,-25}" (they usually don't use space after comma)

and use 'format' in the rest of the code

Reply:
not getting you .. whats meant by  using 'format' ?    is the suggestion to use string.Format  somehow ..

---
In reply to: [271455734](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271455734) [](ancestors = 271455734)

Reply:
used control+k+d  to fix format (if thats what u meant)

---
In reply to: [271462084](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271462084) [](ancestors = 271462084,271455734)

Reply:
Ivan meant to define the "{0,-25}" as a variable, rather than repeating it, and reuse the variable below. 

---
In reply to: [271465497](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271465497) [](ancestors = 271465497,271462084,271455734)